systest/Makefile

```diff
@@ -130,21 +130,11 @@ cleanup_failed_tlc:
 
 # BRANCH MAKE TARGETS:
 #
-# These make targets invoke OTHER targets defined in this file.  They are not
-# used to invoke particular tests, rather they manage test infrastructure, and
-# conform to the standard interface expected by buildbot.
-#
-# Run all tests, this is what buildbot calls: make functest
-# These are all of the functests that we want to run as part of bbot job
-# We ignore the return value so that we run all of the tests
-functest: install_tlc install_test_infra
-	-$(MAKE) -C . tempest_12.1.1_undercloud
-
 # Not using the tempest TLC files for liberty because they install barbican
 # which causes the tests to lockup/hang
 
 # Tempest Tests for 12.1.x undercloud VE deployment
-tempest_12.1.1_undercloud:
+tempest_12.1.1_undercloud_vxlan:
```

Comment:
The "RESULTS_DIR" variable should have been changed for this job (by dropping the "$(CHANGESOURCE)" path segment).